pkgs/tools/misc/dasht/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, makeWrapper
+, coreutils
+, gnused
+, sqlite
+, wget
+, w3m
+, socat
+, gawk
+}:
+
+stdenv.mkDerivation rec {
+  version = "2.3.0";
+  name    = "dasht-${version}";
+
+  src = fetchFromGitHub {
+    owner  = "sunaku";
+    repo   = "dasht";
+    rev    = "v${version}";
+    sha256 = "0d0pcjalba58nvxdgn39m4b6n9ifajf3ygyjaqgvzwxzgpzw0a60";
+  };
+
+  deps = lib.makeBinPath [ coreutils gnused sqlite wget w3m socat gawk (placeholder "out") ];
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+    cp bin/* $out/bin/
+
+    mkdir -p $out/share/man1
```

Comment:
the manpages should be installed in `$out/share/man/man1`, sorry if I was unclear.